electron/legendary/games.ts

```diff
@@ -215,12 +217,45 @@ class LegendaryGame extends Game {
     const logPath = `"${heroicGamesConfigPath}${this.appName}.log"`
     const writeLog = isWindows ? `2>&1 > ${logPath}` : `|& tee ${logPath}`
 
+    const gameInfo = await this.getGameInfo()
     const command = `${legendaryBin} install ${this.appName} --base-path ${path} ${workers} -y ${writeLog}`
     console.log(`Installing ${this.appName} with:`, command)
     try {
       LegendaryLibrary.get().installState(this.appName, true)
       return await execAsync(command, execOptions).then((v) => {
         this.state.status = 'done'
+        // Add shortcut to desktop folder
+        const desktopFolder = app.getPath('desktop')
+        switch(process.platform) {
+        case 'linux': {
+          const linuxShortcut = `[Desktop Entry]
+Name=${gameInfo.title}
+Exec=xdg-open heroic://launch/${gameInfo.app_name}
+Terminal=false
+Type=Application
+Icon=${gameInfo.art_square}
+Categories=Game;
+`
+          const enabledInDesktop = GlobalConfig.get().config.enableDesktopShortcutsOnDesktop
+          const enabledInStartMenu = GlobalConfig.get().config.enableDesktopShortcutsOnStartMenu
+
+          if (enabledInDesktop === true || enabledInDesktop === undefined) {
```

Comment:
In case it's true OR undefined (it hasn't been defined), create a desktop launcher.

It could be reduced to the variable name too, but that's dangerous in this case. I need type precision.